website/docs/reference/resource-properties/description.md

```diff
@@ -145,6 +146,33 @@ macros:
 
 </TabItem>
 
+<TabItem value="singular_data_tests">
+
+<VersionBlock firstVersion="1.9">
+
+<File name='tests/singular/schema.yml'>
+
+```yml
+version: 2
+
+data_tests:
+  - name: singular_data_test_name
+    description: markdown_string
+
+```
+
+</File>
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.8">
+
+The `description` property is available for [singular data tests](/build/data-tests#document-singular-data-tests) beginning in dbt v1.9.
```

Comment:
@dbeatty10 this is linking to a page not found -- shouldn't this pr include a 'document singular data tests' header in the 'documentation' page for top level keys?

Reply:
Good catch! I'll fix this

Reply:
Thx for fixing this! 💪